services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/Hakemusalue.kt

```diff
@@ -41,3 +41,10 @@ data class Tyoalue(
     val area: Double,
     val tormaystarkasteluTulos: TormaystarkasteluTulos?,
 )
+
+fun List<KaivuilmoitusAlue>?.toPostalAddress(): PostalAddress? =
```

Comment:
I think the `toXxx()` naming should be used only when it's implicitly clear why we have this conversion. For example, converting a domain class to a DTO is pretty self-explanatory. I think something like `combineAddresses()` would be more descriptive here.

Reply:
I renamed this as `combinedAddress()`.